src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promiseMessage = (text, style = 'message') => {
```

Comment:
```suggestion
const printMessage = (text, style = 'message') => {
```
![image](https://github.com/mate-academy/js_promise_basic_DOM/assets/96409563/87f0e948-f20e-49da-93bb-1e21705ec4d1)